sematic/ui/packages/main/src/pipelines/graph/RunNode.tsx

```diff
@@ -31,6 +32,12 @@ function getColor(futureState: string, theme: Theme): PaletteColor {
   };
 }
 
+const StyledAlert = styled(Alert)`
+  & .${alertClasses.message} {
+    overflow-y: hidden;
```

Comment:
Interesting! Why haven't we seen this manifest before?
Can the size of the box be sized to fit the text (with a max size) instead of clipping the text?

Reply:
Good question. I am not sure. One baseless guess is that some dependency library (eg MUI) has changed during this time. (the file was only changed 2 months ago)

Investigating might take a significant amount of time. But if it is necessary, please let me know.

Reply:
Nah, that's fine. :shipit: